### PR TITLE
Very basic client set up with react

### PR DESCRIPTION
**Pull request by AshbyGunter**
All elements rendered with react. 
Everything is static placeholder text thus far.
Supporting file structure and files created.
Introduced airbnb llinting.
Ran linting rules on all files and fixed all errors, but kept some server logging warnings.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -6,5 +6,6 @@
  */
 
 module.exports = {
-  extends: './node_modules/eslint-config-hackreactor/index.js'
+  // extends: './node_modules/eslint-config-hackreactor/index.js'
```
Comment (Melvin-Viana) [inline .eslintrc.js:9]: I think you could remove this line. I don't think we'll need to use the hackreactor styling guide for this project.

**Comment by Melvin-Viana** [comment]
I Like how you documented your React Components at the end of the files. Airbnb has a lot of documentation on their JS styling guide. 

I kind of skimmed through these changes seeing that it's all just code styling. It looks good to me.

File: .eslintrc.js (hunk 1 above)
Comment (AshbyGunter) [reply .eslintrc.js:9]: good call, thanks.

**Comment by AshbyGunter** [comment]
Thanks! I found it helped as I was setting up to help me keep in mind my structure and the goals for each component. I'll keep them there for now, but need to keep in mind to alter them if I change any of that structure.

**Pull request merged**